settings.nix

```diff
@@ -2010,6 +2073,11 @@ with docs.lib; rec {
             })) [
             (mapAttrsToList leaf cfg.action)
           ];
+
+        workspace = cfg:
+          node "workspace" cfg.name [
+            (nullable leaf "open-on-output" cfg.open-on-output or null)
```

Comment:
won't this also always be null anyways? so the right half of `or null` here is unreachable, though it will work as expected.

the other places in the settings module where `or null` is used is when accessing properties of `variant`s, because they will not set uninhabited variants to null

Reply:
Oops. That's a remnant from a previous implementation. Fixing in a moment.